### DISCUSSION TITLE
The current state-of-the-art for single-card training of models is at what level? 目前给出的单卡训练模型方案大概是什么水平？

### DISCUSSION THREAD

**Discussion by ryan-utopia** [hpcaitech/ColossalAI#3210] [Community | General]
For example, how long would it take to train a model （GPT-2 or PaLM） using a single A100 card? 比如使用一张A100训练的话大概要训练多久呢？

**Comment by binmakeswell** [comment]
Hi @JacketChenlll There is no standard answer to this as there are many influencing factors such as hardware configuration, model size, parameter configuration, dataset size, batchsize, etc.
